packages/spec-api/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@typespec/spec-api",
+  "version": "0.1.0",
+  "description": "Spec api to implement mock api",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "watch": "tsc -p ./tsconfig.build.json --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "clean": "rimraf dist/ temp/",
+    "lint": "eslint . --max-warnings=0",
+    "lint:fix": "eslint . --fix",
+    "test": "mocha"
+  },
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "homepage": "https://github.com/microsoft/typespec#readme",
+  "dependencies": {
+    "body-parser": "^1.20.2",
+    "deep-equal": "^2.2.0",
+    "express": "^4.19.2",
+    "express-promise-router": "^4.1.1",
+    "glob": "^10.2.2",
```

Comment:
maybe good to update to globby like other packages in this repo

Reply:
This package does not need glob at all. So, I have removed it. But, replaced the glob with globby in the lib package. Thanks